formulador-lite/drawing.lisp

```diff
@@ -0,0 +1,31 @@
+;;;;drawing.lisp
+
+(in-package :formulador-lite)
+
+
+;;;;;------------------------------------------------------------------------
+;;;;;The simple drawing command: (simple-draw "1+2")
+;;;;;------------------------------------------------------------------------
+(defun simple-draw (formula-string)
+  "Given a formula-string, returns pretty-printed formulador formula."
+  (formulador::draw
+   (eval (first (block-cycle (block-list (lex-line formula-string)))))))
```

Comment:
you can avoid `eval` by building the structures in the first place, instead of making quoted lists.